packages/iov-lisk/src/liskconnection.ts

```diff
@@ -198,8 +234,40 @@ export class LiskConnection implements BcpConnection {
     throw new Error("Not implemented");
   }
 
-  public searchTx(_: BcpTxQuery): Promise<ReadonlyArray<ConfirmedTransaction>> {
-    throw new Error("Not implemented");
+  public async searchTx(query: BcpTxQuery): Promise<ReadonlyArray<ConfirmedTransaction>> {
+    if (query.height || query.minHeight || query.maxHeight || query.tags.length) {
+      throw new Error("Query by height, minHeight, maxHeight, tags not supported");
+    }
+
+    if (query.hash) {
```

Comment:
Nice to add this support to Lisk.
There is no indexing service to allow any other queries, is there?

Reply:
> There is no indexing service to allow any other queries, is there?

We can query all kinds of things and probably most of the interface can be supported. But I only implemented what I need now, which already blows up this PR enough. But we can make it happen as soon as we need it :)